src/Workspaces/Core/Portable/Workspace/Solution/SolutionInfo.cs

```diff
@@ -72,6 +84,18 @@ private SolutionInfo(SolutionAttributes attributes, IReadOnlyList<ProjectInfo> p
         string? filePath = null,
         IEnumerable<ProjectInfo>? projects = null,
         IEnumerable<AnalyzerReference>? analyzerReferences = null)
+        => Create(id, version, filePath, projects, analyzerReferences, ImmutableDictionary<string, StructuredAnalyzerConfigOptions>.Empty);
```

Comment:
can be in followup, but i would def like a stronger named type here.

Reply:
Sounds good. Will do as follow up.